x-pack/plugins/cross_cluster_replication/public/app/services/api.js

```diff
@@ -19,6 +20,9 @@ export function setHttpClient(client) {
 
 // ---
 
+
 export const loadAutoFollowPatterns = async () => {
-  return await httpClient.get(`${apiPrefix}/auto_follow_patterns`).then(response => response.data);
+  return await httpClient.get(`${apiPrefix}/auto_follow_patterns`)
+    .then(wait(1000))
```

Comment:
The way this wait API works is kind of weird, as reading just this file it seems we're just always waiting, where in reality we only do so if simulate http latency is set to true.  I'm not really sure that this code is good to leave in.  If we're testing out loading screens we could do that in a way that wouldn't leave code that is confusing in after the testing. (fiddler for example https://www.telerik.com/fiddler)

Reply:
You're totally right, will use other tool for this purpose